samples/server/petstore/spring-boot-nullable-set/src/main/java/org/openapitools/model/ObjectWithUniqueItems.java

```diff
@@ -13,6 +13,8 @@
 import java.util.Set;
 import org.openapitools.jackson.nullable.JsonNullable;
 import org.springframework.format.annotation.DateTimeFormat;
+import com.fasterxml.jackson.annotation.JsonIgnore;
```

Comment:
Nit: import seems unused?

Reply:
Hi, this has been resolved in latest commit